src/main/java/winstone/Launcher.java

```diff
@@ -337,9 +305,6 @@ public void shutdown() {
             Logger.log(Logger.INFO, RESOURCES, "Launcher.FailedShutdown", e);
         }
 
-        // Release all listeners/pools/webapps
-        this.threadPool.shutdown();
```

Comment:
Do we still need to clean up the thread pool, or does Jetty handle it now?

Reply:
Jetty has his own lifecyle see doSop here https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/jetty-util/src/main/java/org/eclipse/jetty/util/thread/QueuedThreadPool.java#L150